client/payment-details/summary/index.tsx

```diff
@@ -57,7 +77,11 @@ const composePaymentSummaryItems = ( { charge }: { charge: Charge } ) =>
 			title: __( 'Channel', 'woocommerce-payments' ),
 			content: (
 				<span>
-					{ getChargeChannel( charge.payment_method_details?.type ) }
+					{ isTapToPay( metadata?.reader_model || '' )
```

Comment:
I guess we can just remove the `|| ''` given it will fail the comparison inside the `isTapToPay` anyway.
```suggestion
					{ isTapToPay( metadata?.reader_model )
```

Reply:
Updated!